File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.js
function solution(n) {
    const mid = parseInt(n/2)
    let subak = ''
    
    if(n % 2 == 1) {
        for(let i=0; i<mid; i++) {
            subak += "수박"
        }
        subak += "수"
    }
    
    else {
        for(let i=0; i<mid; i++) {
            subak += "수박";
        }
    }
    return subak;
}

File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int n) {
    string answer = "";
    
    int watermelon = n / 2;
    int su = n % 2;
    for(int i=0; i<watermelon; i++) {
        answer += "수박";
    }
    
    return su > 0 ? answer + "수" : answer;
}